frontend/src/components/Dialogs/InputDialog.vue

```diff
@@ -66,10 +71,9 @@ export default {
   padding: var(--p-16);
   display: flex;
   flex-direction: column;
-  min-width: 400px;
-  max-width: auto;
+  min-width: 25rem;
   min-height: auto;
-  max-height: auto;
+  max-height: none;
```

Comment:
The default value for `max-height` is none, remove excess CSS declarations

Reply:
removed